### DISCUSSION TITLE
CoPilot Was wr

### DISCUSSION THREAD

**Discussion by dustindodson** [community/community#39677] [Copilot Conversations]
### GitHub CoPilot

VS2022 - Bug

### Giving Up?

![Giving Up](https://user-images.githubusercontent.com/40292725/203103742-8c9785d2-391e-4058-99f8-0b760dd85b7d.jpg)

So are we just giving up?

This was working when I bought Co-Pilot.... 

I guess the money you can make charging companies is worth more to you than customer..... Sorry SUPPORTER satisfaction.. 

It makes me sick to my stomach that this was working... and  now it doesn't not even 3 weeks after getting my money.

It makes me even more ill to know that behind the same corporate firewall.... same computer, same user... This works perfectly in VS Code... 

I would ask for my money back... but I know deep in my heart, that it would be a cold day in hell before I got my money back.... 

Me the individual..... SUPPORTER that $100s actually means something to... could have paid bills with it... put food in my child's mouth....

I know that this will more than likely get censored from your site.... but YOU will read it....

So SHAME!!.   Shame on you for disgracing the spirit of open source code and it's supporters...

Shame on you for swindling tons of your fellow developers after you have their money...

Shame on you for taking food out of my child's mouth........

Shame....

I don't know how you sleep at night.

**Comment by davecheney** [comment]
@dustindodson i’m sorry that you are experiencing difficulties. I think I understand the problem you are describing, but I’d like to repeat what I read to make sure were aligned;

Are you saying that you have purchased copilot as an individual? 

If so, you tried to use copilot while vscode was signed in to your companies EMU GitHub identity. Support for EMU accounts will be available from December 7, and if you haven’t already have someone from your company reach out to their GitHub sales rep to join the dots. 

If you have a GitHub for individuals account, 
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

lastly, please accept my apologies if I didn’t not answer your question accurately. I understand that you are disappointed in us, and if you can give some more information about what you tried to do, and what happened when you tried I’m sure we can work to a solution.

**Comment by dustindodson** [comment]
Single license...
It's the self signing cert issue
VSCode was not installed at the time of it starting not to work.

I'm signed into a personal account in both IDEs

Thanks for the response.

**Reply by davecheney** [reply]
I believe that self signed certs _are_ supported in VSCode and VS (not code), however sadly I cannot find the documentation on this feature at the moment. I've asked the development teams to link me to the documentation and I'll update this discussion when they provide it.

**Reply by dustindodson** [reply]
Exactly why I wrote the post.....

That was not the case when I bought it... 

Had I known that was going to be the case.... Well let's be honest..... POLICY,   I never would have bought the product.

Because we don't use VSCode at my work.

And I'm quite sure I'm not the only one who feels that way....

Anyone know a good class action lawyer??? 

Thanks anyway...

**Reply by davecheney** [reply]
I'm sorry that you are having difficulties. Please contact support if you wish to request a refund, https://support.github.com/contact
